expyfun/_utils.py

```diff
@@ -37,6 +37,14 @@
     pass
 
 try:
+    from pyglet.media import avbin
+    del avbin
+except ImportError:
+    has_avbin = False
+else:
+    has_avbin = True
```

Comment:
So far we've managed to avoid importing `Pyglet` at all until a function that needs it is used. Could you make the decorator actually do this check internally? That way, this import is done only when running tests, not when running `expyfun`.